src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomUap.java

```diff
@@ -63,15 +63,15 @@ public void adsFandomHubUapDesktop(String pageType, String pageName, long atfId,
   )
   @Test(
       dataProviderClass = FandomAdsDataProvider.class,
-      dataProvider = "fandomHubUapPage",
+      dataProvider = "fandomTopicPage",
       groups = {"AdsFandomUapMobile"}
   )
   @RelatedIssue(issueID = "ADEN-4339")
-  public void adsFandomHubUapMobile(String pageType, String pageName, long atfId, long btfId) {
+  public void adsFandomUapOnTopicMobile(String pageType, String pageName, long atfId, long btfId) {
     AdsFandomObject fandomPage = loadPage(pageName, pageType);
     fandomPage.triggerOnScrollSlots();
     verifyUapAtf(atfId, AdsFandomContent.TOP_BOXAD, fandomPage);
-    verifyUapBtf(btfId, AdsFandomContent.INCONTENT_BOXAD_MOBILE, AdsFandomContent.BOTTOM_LEADERBOARD, fandomPage);
+    verifyUapBtf(btfId, AdsFandomContent.FEED_BOXAD, AdsFandomContent.BOTTOM_LEADERBOARD, fandomPage);
   }
 
   private void verifyUapAtf(long atfId, String slotName, AdsFandomObject fandomPage) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyUapAtf" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)